queries/koto/highlights.scm

```diff
@@ -26,10 +26,9 @@
   "or"
 ] @keyword.operator
 
-[
-  "return"
-  "yield"
-] @keyword.return
+"return" @keyword.return
+
+"yield" @keyword.coroutine
```

Comment:
I think you can revert this change if `yield` is only used in iterators. If you know it can be a part of a coroutine call then please only narrow coroutine to that.

Reply:
Ok that makes sense. `yield` will be used in coroutines later on, but for now they're only used in custom iterators. When things change I'll update the capture used here.